wallet-access/.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["Vue.volar", "Vue.vscode-typescript-vue-plugin"]
+}
```

Comment:
remove .vscode from dir and add it to the gitignore, better to have the one gitignore for all repo in the root dir